.projenrc.js

```diff
@@ -30,6 +37,15 @@ project.eslint.addRules(
   },
 );
 
+project.gitignore.exclude('.vscode/');
+project.gitignore.exclude('*.d.ts');
+project.gitignore.exclude('*.js');
+
+project.npmignore.exclude('!*.js');
+project.npmignore.exclude('*.ts');
+project.npmignore.exclude('!*.d.ts');
+
+/*
```

Comment:
I think we would still like examples to compile. Just no need for the node modules hack

Reply:
removed for now; was seeing strange behavior but maybe that has cleared up as a result of some other changes.